plugins/nessus.rb

```diff
@@ -370,15 +423,23 @@ def cmd_nessus_save(*args)
       end
 
       def cmd_nessus_server_properties(*args)
-        if args[0] == "-h"
-          print_status("nessus_server_feed")
-          print_status("Example:> nessus_server_feed")
-          print_status()
-          print_status("Returns information about the feed type and server version.")
-          return
+        search_term = nil
+        while (arg = args.shift)
+          case arg
+          when '-h', '--help'
+            print_status("nessus_server_feed")
+            print_status("Example:> nessus_server_feed -S searchterm")
```

Comment:
This was wrong before and it is wrong here as well. The command is nessus_server_properties but the example is nessus_server_feed. We need to change either the command or the example description.